.github/workflows/npm-publish.yml

```diff
@@ -20,3 +21,42 @@ jobs:
       - run: npm publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.npm_token }}
+
+  publish-docker:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
A comment of why this is added would be nice, it is a requirement of buildx right?